src/sync/test.rs

```diff
@@ -33,22 +33,10 @@ where
     T: Serialize + DeserializeOwned + Unpin + Debug + Send + Sync,
 {
     let coll = client.database(db_name).collection(coll_name);
-    drop_collection(&coll);
+    coll.drop(None).unwrap();
     coll
 }
 
-pub fn drop_collection<T>(coll: &Collection<T>)
```

Comment:
This is no longer necessary since our `drop` helper ignores ns not found errors already.